src/docs/asciidoc/en/rlsnotes/rlsnotes30/_utilities-30.adoc

```diff
@@ -285,6 +285,13 @@ _gfix_ has a new switch `-NoLinger` to provide a one-off override to the `LINGER
 
 For information regarding `LINGER`, see <<rnfb30-ddl-enhance-linger,the write-up in the DDL chapter>>.
 
+[[rnfb30-util-gfix-icu]]
+=== -icu switch
+
+_gfix_ has a new switch `-icu` to update ICU-dependent collations and rebuild dependent indices.
+
+This can be used to make indices usable again when moving databases -- without backup and restore -- between Firebird installations using different ICU versions, or when the ICU version used by a Firebird install is updated.
```

Comment:
Why "by a Firebird install"? Install itself does not use ICU. Saying "when the ICU version used by a Firebird is updated" appears better.

Reply:
With "Firebird install" I meant an installed Firebird instance, not the installation process. I'll tweak the wording.